generator/ServiceClientGeneratorLib/Generators/Marshallers/JsonRPCStructureMarshaller.tt

```diff
@@ -54,18 +54,18 @@ namespace <#=        this.Config.Namespace #>.Model.Internal.MarshallTransformat
 			string memberProperty = variableName + "." + member.PropertyName + (member.IsNullable ? ".Value" : string.Empty);
 			if(member.IsStructure || member.IsList || member.IsMap)
 			{
-				this.ProcessStructure(level, variableName + "." + member.PropertyName, member.Shape);
+				this.ProcessStructure(level + 1, variableName + "." + member.PropertyName, member.Shape);
```

Comment:
I don't understand this change, the generated files looks fine but I want to understand how it used to work before without the +1?